kmk/extensions/peg_oled_display.py

```diff
@@ -98,16 +97,17 @@ def renderOledTextLayer(self, layer):
                 y=25,
             )
         )
+        self._display.show(splash)

```

Comment:
Moves `self._display.show(splash)` to after setting up the `displayio.Group()` so that content no longer flashes/flickers when changing display content based on current layer.